src/current/_includes/v24.1/known-limitations/aost-limitations.md

```diff
@@ -0,0 +1 @@
+CockroachDB does not support placeholders in [`AS OF SYSTEM TIME`]({% link {{ page.version.version }}/as-of-system-time.md %}). The time value must be a constant value embedded in the SQL string. [#30955](https://github.com/cockroachdb/cockroach/issues/30955)
```

Comment:
When this file is included on the `as-of-system-time.md`, the `AS OF SYSTEM TIME` link just goes to the top of the same page. Do you think it would be better to have:

 ```suggestion
CockroachDB does not support placeholders in {% if page.name == "as-of-system-time.md" %}`AS OF SYSTEM TIME`{% else %}[`AS OF SYSTEM TIME`]({% link {{ page.version.version }}/as-of-system-time.md %}){% endif %}. The time value must be a constant value embedded in the SQL string. [#30955](https://github.com/cockroachdb/cockroach/issues/30955)
```

if you add conditional to this include file, please also add to `create-statistics-aost-limitations.md` and to v24.2 corresponding include files.

Reply:
Good idea -- done.